src/cpx_model.jl

```diff
@@ -240,3 +240,55 @@ function set_terminate(model::Model)
 end
 
 terminate(model::Model) = (model.terminator[1] = 1)
+
+mutable struct ConflictRefinerData
+    stat::Int
+    nrows::Int # number of rows participating in the conflict
+    rowind::Vector{Cint} # index of the rows that participate
+    rowstat::Vector{Cint} # state of the rows that participate
+    ncols::Int # number of columns participating in the conflict
+    colind::Vector{Cint} # index of the columns that participate
+    colstat::Vector{Cint} # state of the columns that participates
+end
+
+function refineconflict(model::Model)
```

Comment:
Please call this `c_api_getconflict` and ad a comment describing how `refineconflict` is always called first.